ref/README.md

```diff
@@ -79,6 +84,27 @@ On many systems, the above will suffice. However, some systems will require you
 find dependencies, particularly if software depencencies are not installed in standard locations.
 See below for more information.
 
+### Building with GPTL profiling support
```

Comment:
I would suggest adding a link to the GPTL documentation for the hardy souls that want to insert their own GPTL calls.